library/psa_crypto.c

```diff
@@ -451,13 +451,6 @@ static psa_status_t prepare_raw_data_slot( psa_key_type_t type,
     switch( type )
     {
         case PSA_KEY_TYPE_RAW_DATA:
-            if( bits == 0 )
-            {
-                raw->bytes = 0;
-                raw->data = NULL;
-                return( PSA_SUCCESS );
-            }
-            break;
```

Comment:
Don't we want to keep `break` here? or is it correct to `/* fallthrough */`?

Reply:
`RAW_DATA`, `HMAC` and `DERIVE` have the same treatment. No fallthrough, just several values with the same handling.